apps/snitch_core/test/data/schema/package_test.exs

```diff
@@ -57,7 +57,7 @@ defmodule Snitch.Data.Schema.PackageTest do
   @update_params %{
     shipping_methods: [],
     tracking: %{id: "some_tracking_id"},
-    state: "ready!",
+    state: "ready",
```

Comment:
@anantanant2015 shouldn't this be atom? if not why not?

Reply:
because when params are casted then they can be either in type integer, string or atom, since in the controller params are received in string form, here are the valid values map
PackageStateEnum.__valid_values__()
[0, 1, 2, 3, 4, 5, 6, :pending, :processing, :fulfilled, :ready, :shipped,
 :delivered, :complete, "complete", "delivered", "fulfilled", "pending",
 "processing", "ready", "shipped"]